Addresses Issues 20 22 41 42 44 45

McFateM opened the pull request:
This PR addresses issues 20, 22, 41, 42, 44 and 45.  All "issue" references have been removed from the code.  The big change here is my abandoning the use of an "import_index" column in favor of a UI element that triggers the insertion of a specially named data column called "imi_import_source_tracking".  The UI to control this exists only for the Google Sheet option.

Three functions in this PR are tagged as "tempory" and are only effective the "restore" function is called from a hook_form_alter implementation.  

My previous two PRs will be closed in favor of this one.

DiegoPino commented:
@McFateM cool. Will give this a spin today and will come back to you if something more is missing or will go and merge if all good. Great contribution!

DiegoPino commented:
@McFateM there are some little issue i need to address. So will merge your code into a new branch (grinnel) and will fix there. Then will incorporate those to the other changes (plupload integration + some fixes on the twig template editor piece). This is great. Will ask you for some info (like why the date(c) as default for the spreadsheet tracking?). All your changes are great improvements to this module!!

McFateM commented:
Thanks Diego.   The date(c) default in that field plays nicely into our workflow.  We capture the value in that field as a MODS extension like so…

<extension>
  <primarySort>99</primarySort>
  <dg_importSource>2017-11-15T09:13 - https://docs.google.com/spreadsheets/d/1wAwJwxOURDMgKoh8SZ8IIpOlrucNaFtbZQK9bj3bqys - 9AM!A1:AK147 | 17</dg_importSource>
  <dg_importIndex>27</dg_importIndex>
</extension>

And the timestamp (default value) at the start of the dg_importSource field is helps identify when the import was prepared.

From: Diego Pino Navarro <notifications@github.com<mailto:notifications@github.com>>
Reply-To: mnylc/islandora_multi_importer <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Sunday, November 19, 2017 at 12:01 PM
To: mnylc/islandora_multi_importer <islandora_multi_importer@noreply.github.com<mailto:islandora_multi_importer@noreply.github.com>>
Cc: Mark McFate <mcfatem@grinnell.edu<mailto:mcfatem@grinnell.edu>>, Mention <mention@noreply.github.com<mailto:mention@noreply.github.com>>
Subject: Re: [mnylc/islandora_multi_importer] Addresses Issues 20 22 41 42 44 45 (#51)


@McFateM<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_mcfatem&d=DwMFaQ&c=HUrdOLg_tCr0UMeDjWLBOM9lLDRpsndbROGxEKQRFzk&r=PQglHQe-EzyZqJOuOVcmU0OZ6bg-89msSPuqyNlQr28&m=SmpltPutAZyVWwaQiJWGXtjLCZyWY76eSqLpyALLh3o&s=Zxs2q5Y6pot888QHL6G9BaiOFQGgSGEXRsWFWs-9wN8&e=> there are some little issue i need to address. So will merge your code into a new branch (grinnel) and will fix there. Then will incorporate those to the other changes (plupload integration + some fixes on the twig template editor piece). This is great. Will ask you for some info (like why the date(c) as default for the spreadsheet tracking?). All your changes are great improvements to this module!!

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_mnylc_islandora-5Fmulti-5Fimporter_pull_51-23issuecomment-2D345536524&d=DwMFaQ&c=HUrdOLg_tCr0UMeDjWLBOM9lLDRpsndbROGxEKQRFzk&r=PQglHQe-EzyZqJOuOVcmU0OZ6bg-89msSPuqyNlQr28&m=SmpltPutAZyVWwaQiJWGXtjLCZyWY76eSqLpyALLh3o&s=Me4OI3F3b7ldqJIB1xsuUF4y52pwET42JqUfstHayPU&e=>, or mute the thread<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_AIFIwVkyRZJE90aTuf14e68iOUstpYUTks5s4G0EgaJpZM4QYrhC&d=DwMFaQ&c=HUrdOLg_tCr0UMeDjWLBOM9lLDRpsndbROGxEKQRFzk&r=PQglHQe-EzyZqJOuOVcmU0OZ6bg-89msSPuqyNlQr28&m=SmpltPutAZyVWwaQiJWGXtjLCZyWY76eSqLpyALLh3o&s=5E5n-EMJwCz4QLD84rRtK7t4cDFwDZYMIoPR1bfbZmk&e=>.
